adagrams/game.py

```diff
@@ -1,13 +1,152 @@
 from random import randint
+    # from tests.test_wave_01 import LETTER_POOL
+
+LETTER_POOL = {
+    'A': 9,
+    'B': 2,
+    'C': 2,
+    'D': 4,
+    'E': 12,
+    'F': 2,
+    'G': 3,
+    'H': 2,
+    'I': 9,
+    'J': 1,
+    'K': 1,
+    'L': 4,
+    'M': 2,
+    'N': 6,
+    'O': 8,
+    'P': 2,
+    'Q': 1,
+    'R': 6,
+    'S': 4,
+    'T': 6,
+    'U': 4,
+    'V': 2,
+    'W': 2,
+    'X': 1,
+    'Y': 2,
+    'Z': 1
+}
 
 def draw_letters():
-    pass
+    letter_pool_list = []
+    for letter, count in LETTER_POOL.items():
+        letter_pool_list.extend([letter] * count)
+        # transfer the dictionary to a 98 letters list to use random.randint
+
+    draw_letter_pool = letter_pool_list.copy()
+    # make a copy list for poping letter to avoid draw a letter and hands repeatly
+    # use index / .pop() use index
+
+    draw_hand = []
+
+    while len(draw_hand) < 10 :
```

Comment:
In Python, we call 10 in this context a "magic number". This means that the number has been hardcoded without much context. If I were a developer unfamiliar with Adagrams, I might not know exactly what that 10 means. We can avoid these magic numbers by making constants with more descriptive names to hold them. In this instance, we know that 10 is the max size of the hand, so we might do something along the lines of:

```suggestion
    MAX_HAND_SIZE = 10
    while len(draw_hand) < MAX_HAND_SIZE :
```